icepyx/tests/test_visualization.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from icepyx.core.visualization import Visualize
+
+
+@pytest.mark.parametrize("product, date_range, bbox, expect", [
+    ('ATL06',['2019-6-15', '2019-7-1'], [-64.5, -66, -63.5, -65], 3240),
+    ('ATL07',['2019-7-1', '2019-8-1'], [-65, -66, -64.5, -65], 7160),
+    ('ATL08',['2019-6-15', '2019-7-1'], [-18, 63, -17, 64], 852),
+    ('ATL10',['2019-8-1', '2019-9-1'], [-64, -67, -60, -60], 7375),
+    ('ATL12',['2019-7-1', '2019-10-1'], [-65.5, -65.5, -64.5, -65], 95),
+    ('ATL13',['2019-6-1', '2019-12-1'], [-75, -51, -74, -50], 20),
```

Comment:
Do you know how OA handles new version releases? I'm concerned these tests could break if a new version of a product is released, but I don't know if that's a thing we can control for or will just have to fix manually.

Reply:
There doesn't seem to be a version option listed at https://openaltimetry.org/data/swagger-ui/#/Public/getATL06DataByDate (or the other ATLAS products). Might just have to update this every so often, There has only been 4 version releases for ATL06 over the past 2 years so shouldn't be too much work :crossed_fingers:.

Reply:
Agree with @weiji14, we could update this when a new version is released.